tests/test_integration.py

```diff
@@ -1315,6 +1315,33 @@ def test_no_emit_warnings_verbose_override():
   assert stderr
 
 
+def test_undeclared_setuptools_import_on_pex_path():
+  with temporary_dir() as td:
+    setuptools_pex = os.path.join(td, 'setuptools.pex')
+    run_pex_command(['setuptools==40.6.3', '-o', setuptools_pex]).assert_success()
```

Comment:
Is there a particular reason for this version? Does it need to be different than what Pex vendors here?

https://github.com/pantsbuild/pex/blob/3a7e4d701f8ace8bd070e53f03cc694e885e52c5/pex/vendor/__init__.py#L74-L80.

If so, I recommend adding a brief `NB` about this.

Reply:
I've only been able to repro this issue with setuptools being provided in this exact way. The version of setuptools doesn't seem to matter, but I wanted to pin it so that we don't have setuptools eventually fixing the bug and silently break the test.

Reply:
That'd be helpful to document, e.g.

```
NB: the specific setuptools does not necessarily matter. We only pin the version to avoid a future version of setuptools potentially fixing this issue and then us no longer checking that Pex is behaving properly for older setuptools versions.
```

Reply:
Done!